package.json

```diff
@@ -33,7 +33,7 @@
 	],
 	"scripts": {
 		"lint": "eslint --max-warnings=0 './src/**/*.{ts,js}'",
-		"prettier": "prettier './**/*.js' './src/**/*.ts' './**/*.md' --write",
+		"prettier": "prettier . --write",
```

Comment:
In Prettier's very early days you had to explicitly specify which files to check/format like this, but since Prettier 2.0 you can just pass `.` to expand the current directory, and Prettier will check+format all _compatible_ files (and ignore any incompatible ones).